contracts/src/v2/IGateway.sol

```diff
@@ -29,9 +29,7 @@ interface IGatewayV2 {
     event AgentCreated(bytes32 agentID, address agent);
 
     /// Emitted when inbound message has been dispatched
-    event InboundMessageDispatched(
-        uint64 indexed nonce, bytes32 topic, bool success, bytes32 rewardAddress
-    );
+    event InboundMessageDispatched(uint64 indexed nonce, bool success, bytes32 rewardAddress);
```

Comment:
Adding the topic ID here is good for the indexer, but we missed the change on Substrate side. Otherwise, the `submit_delivery_proof` extrinsic will fail.

Since the upstream PR is almost finalized, just revert it here.

Reply:
Reverted